api/turing/cluster/pdb.go

```diff
@@ -0,0 +1,51 @@
+package cluster
+
+import (
+	"fmt"
+
+	apipolicyv1 "k8s.io/api/policy/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+type PodDisruptionBudget struct {
+	Name           string                `json:"name"`
+	Namespace      string                `json:"namespace"`
+	Labels         map[string]string     `json:"labels"`
+	MaxUnavailable string                `json:"max_unavailable"`
+	MinAvailable   string                `json:"min_available"`
+	Selector       *metav1.LabelSelector `json:"selector"`
+}
+
+func (cfg PodDisruptionBudget) BuildPodDisruptionBudget() (*apipolicyv1.PodDisruptionBudget, error) {
+	if cfg.MaxUnavailable == "" && cfg.MinAvailable == "" {
```

Comment:
Since this check will only kick in much later (when user attempts to deploy a router), can we move it to the config validator instead? https://github.com/caraml-dev/turing/blob/main/api/turing/config/config.go#L619

This way, if one of these 2 values is not set, Turing app can't start up.